.gitignore

```diff
@@ -48,3 +48,5 @@ app.*.map.json
 conf/breez.conf
 build-output/
 google-services.json
+ios/Runner/GoogleService-Info.plist
+build-ios.sh
```

Comment:
Is the extension wrong on this file or is it different from build-ios.yml?

Reply:
I have a script for building in my computer so just excluded it.